tests/sample_data/track_files/csv_data/aircraft/OwnPos_UC_MissingData.csv

```diff
@@ -0,0 +1,2 @@
+Date(Uk),Time(Z),Lat(DD:MM.MM),Long(DDD:MM.MM),Lat(DegN),Long(DegE),Altitude(ft),GndSpeed(kts),Course(deg)
```

Comment:
@SamWilliams-dev - this file is called `missing-data`, but I can't see which field is missing.  It appears to be identical to `OwnPos_UC.csv`.

Reply:
Is this file being deleted now, as we're dealing with the missing/invalid data tests in a different way?

Reply:
Deleted this file now.